convert2rhel/backup.py

```diff
@@ -339,6 +339,78 @@
         super(RestorableRpmKey, self).restore()
 
 
+class NewRestorableFile(RestorableChange):
+    def __init__(self, filepath):
+        super(NewRestorableFile, self).__init__()
+        self.filepath = filepath
+
+    def enable(self):
+        """Save current version of a file"""
+        # Prevent multiple backup
+        if self.enabled:
+            return
+
+        loggerinst.info("Backing up %s." % self.filepath)
+        if os.path.isfile(self.filepath):
+            try:
+                loggerinst.debug("Copying %s to %s." % (self.filepath, BACKUP_DIR))
+                shutil.copy2(self.filepath, BACKUP_DIR)
+
+            except (OSError, IOError) as err:
+                # IOError for py2 and OSError for py3
+                loggerinst.critical("Error(%s): %s" % (err.errno, err.strerror))
+        else:
+            loggerinst.info("Can't find %s.", self.filepath)
+
+        # Set the enabled value
+        super(NewRestorableFile, self).enable()
+
+    def restore(self, rollback=True):
+        """Restore a previously backed up file"""
+        # We do not have backup
+        if not self.enabled:
+            loggerinst.info("%s hasn't been backed up." % self.filepath)
+            return
+
+        backup_filepath = os.path.join(BACKUP_DIR, os.path.basename(self.filepath))
+        if rollback:
+            loggerinst.task("Rollback: Restore %s from backup" % self.filepath)
+        else:
+            loggerinst.info("Restoring %s from backup" % self.filepath)
+
+        # if not os.path.isfile(backup_filepath):
+
+        #    return
```

Comment:
It's leftover from the legacy one, this check is moved to the beginning of the restore method - check if enabled: https://github.com/oamg/convert2rhel/pull/935/files#diff-8f4622020cde9a253bc82f69bf2b88910bcc5a84943db3c7d3a435becedfd4d5R371

I'll remove this, I left it there for not forgetting to this case. You can check if substitute with enabled is fine.

How it works now:
When we try to backup file which is not present, the enabled is set to `True` to prevent second call of this. Then while restoring, the enabled is set to `True` so the `Error(%s): %s` is printed. 

As I think about it now, it might be better for the user to read `%s hasn't been backed up.` than the error. So I might use this part of code for this